plugins/metrics/scheduler.go

```diff
@@ -0,0 +1,102 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/iotaledger/goshimmer/packages/tangle/schedulerutils"
+
+	"github.com/iotaledger/hive.go/identity"
+	"github.com/iotaledger/hive.go/syncutils"
+)
+
+var (
+	// schedulerRate rate at which messages are scheduled.
+	schedulerRate time.Duration
+
+	// readyMessagesCount number of ready messages in the scheduler buffer.
+	readyMessagesCount int
+
+	// totalMessagesCount number of  messages in the scheduler buffer.
+	totalMessagesCount int
+
+	// bufferSize number of bytes waiting to be scheduled.
+	bufferSize int
+
+	// maxBufferSize maximum number of bytes that can be stored in the buffer.
+	maxBufferSize int
+
+	// nodeQueueSizes current size of each node's queue.
+	nodeQueueSizes map[identity.ID]int
+	// nodeQueueSizes current amount of aMana of each node in the queue.
+	nodeAccessMana *schedulerutils.AccessManaCache
```

Comment:
do we need to expose in the metrics the actual AceeManaCacheObject?

it will be more secure if the metrics module can jsut retrieve infor from it but not have real access to it

we can argue that we know what we're doing and we're not writing production code so it isn't so bad

Reply:
I saw that your PR with the remote metrics does it in a better way, so I thought that we can clean this up when we both merge our changes, as it will most likely cause some conflicts anyway.